pkgs/development/libraries/libimobiledevice-glue/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkg-config
+, libplist
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libimobiledevice-glue";
+  version = "unstable-2022-05-22";
+
+  src = fetchFromGitHub {
+    owner = "libimobiledevice";
+    repo = pname;
+    rev = "d2ff7969dcd0a12e4f18f63dab03e6cd03054fcb";
+    hash = "sha256-BAdpJK6/iUKCNYLaCJQo0VK63AdIafO8wGbNhnvEc/o=";
+  };
+
+  outputs = [ "out" "dev" ];
```

Comment:
We prefer having outputs between `version` and `src` to make it immediately obvious that this is a multiple output derivation, see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.